.github/workflows/kubeapps-general.yaml

```diff
@@ -198,9 +198,10 @@ jobs:
       - setup
     env:
       # Note that the max old space setting is per worker, so running the tests
-      # with 4 workers on a 4Gb (free plan) needs 1Gb of max old space. Forcing
-      # garbage collection to start earlier with 512M per worker.
-      NODE_OPTIONS: "--max-old-space-size=512"
+      # with 4 workers on a 7Gb (free plan) needs 1.75Gb of max old space. Forcing
+      # garbage collection to start earlier with 1024M per worker.
+      # See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+      NODE_OPTIONS: "--max-old-space-size=1024"
```

Comment:
Since GHA now has 7GB of mem, I have updated the settings accordingly (I ran into some OOM errors today).